### PR TITLE
Cortx 33258 part2

### PR DESCRIPTION
**Pull request by sonalk0209**
# Problem Statement
- 2 tests which are read only scope tests, optimized and added DTM flag

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tests/ha/single_pod_restart/test_data_pod_restart.py [hunk 1]
```diff
@@ -2822,90 +2820,64 @@ def test_reads_after_pod_restart_ros(self):
...
+            assert_utils.assert_true(resp[0], resp[1])
         resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
                                                     log_prefix=self.test_prefix, skipcleanup=True,
                                                     setup_s3bench=False)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 10: Performed WRITEs/READs/Verify with variable sizes objects.")
+        LOGGER.info("Step 6: Performed WRITEs/READs/Verify with variable sizes objects.")
         LOGGER.info("ENDED: Test to verify READs after data pod restart.")
 
     # pylint: disable=C0321
```
Comment (ankita-93) [inline tests/ha/single_pod_restart/test_data_pod_restart.py:2883]: Please check if we need this warning or not.

**Review by ankita-93 — APPROVED** [review]

File: tests/ha/single_pod_restart/test_data_pod_restart.py [hunk 2]
```diff
@@ -2931,84 +2901,48 @@ def test_continuous_reads_during_pod_restart_ros(self):
...
         thread = threading.Thread(target=self.ha_obj.event_s3_operation,
                                   args=(event,), kwargs=args)
         thread.daemon = True  # Daemonize thread
         thread.start()
         # TODO Need to update timing once we get stability in degraded IOs performance
         time.sleep(HA_CFG["common_params"]["degraded_wait_delay"])
-        LOGGER.info("Step 6: Successfully started READs and verified DI on the written data in "
+        LOGGER.info("Step 3: Successfully started READs and verified DI on the written data in "
                     "background")
-
```
Comment (ApurwaMohite) [inline tests/ha/single_pod_restart/test_data_pod_restart.py:2993]: So because this is ROS test, we are not doing any writes in degraded mode right?

**Review by ApurwaMohite — APPROVED** [review]

**Review by rahul-hatwar — APPROVED** [review]
Looks Good.

**Comment by sonalk0209** [comment]
[pr1761.log](https://github.com/Seagate/cortx-test/files/9131407/pr1761.log)
collection log

File: tests/ha/single_pod_restart/test_data_pod_restart.py [hunk 3]
```diff
@@ -3033,16 +2967,22 @@ def test_continuous_reads_during_pod_restart_ros(self):
         resp = self.ha_obj.check_s3bench_log(file_paths=bkgrd_logs)
         assert_utils.assert_false(len(resp[1]), "Background Logs which contain failures:"
                                                 f" {resp[1]}")
-        LOGGER.info(
-            "Step 6: Successfully completed READs and verified DI on the written data in "
-            "background during pod restart using %s method", self.restore_method)
-
-        LOGGER.info("Step 9: Create multiple buckets and run IOs")
-        self.test_prefix = 'test-36004-1'
+        LOGGER.info("Step 6: Successfully completed READs and verified DI on the written data in "
```
Comment (ravindrachoudhari) [inline tests/ha/single_pod_restart/test_data_pod_restart.py:2970]: Is step 5 missing ?

Comment (sonalk0209) [reply tests/ha/single_pod_restart/test_data_pod_restart.py:2970]: good catch let me check

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**